targets/targets.json

```diff
@@ -4780,6 +4781,7 @@
         "inherits": ["MODULE_UBLOX_ODIN_W2"],
         "supported_form_factors": ["ARDUINO"],
         "release_versions": ["5"],
+        "macros": ["MBED_TICKLESS"],
```

Comment:
I think this should be macros_add instead.

Reply:
Fixed